static/metricsummary.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Summary Table</title>
```

Comment:
1. Put the title as `SigLens`.
2. Look at other HTML files and add similar copyright comments at the top, as well as other gtags and meta tags.
3. Rename `metricssummary.html` to `metric-summary.html` as we use kebab case in other HTML files. Also, rename the JS file to `metric-summary.js`.
4. Do not add a separate CSS file; move everything to the siglens.css file.